modules/social_features/social_event/templates/node--event--email-card.html.twig

```diff
@@ -65,7 +65,7 @@
 <div {{ attributes }}>
   <div class='email_card card__node-email'>
     <div class='card__node-email--header'>
-      <div class='card__node-email--title'>{{ node.type.entity.label() }}</div>
+      <div class='card__node-email--title'>{{ node.type.entity.label()|t }}</div>
```

Comment:
@robertragas @vcsvinicius It's not legal to pass a non-static string to the translation filter and this is unlikely to work unless the whatever comes out of `label` happens to be already in Drupal's translation memory from somewhere else.